_posts/2021-03-31-cockpit-241.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Cockpit 241
+author: skobyda
+date: 2021-03-31
+tags: cockpit linux
+slug: cockpit-241
+category: release
+summary: Cockpit with various improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 241.
+
+## Kdump: Redesign the page
```

Comment:
```suggestion
## Kdump: Beautification and alignment fixes
```

Reply:
I like beautification. Even if it could be cheesy, writing cheesy things into patch and release notes is very popular in IT field I think :D